test/e2e/search-types.spec.js

```diff
@@ -74,10 +74,21 @@ async function checkSearchMetadata(page, searchType) {
   }
 }
 
+async function checkPageMeta(page, searchType) {
+  const sep = searchType.id === 'all' ? '|' : '-'
```

Comment:
Okay, let's definitely change the separator, as identified above, so we can eliminate this logic.